pxtblocks/blocklyloader.ts

```diff
@@ -11,7 +11,9 @@ namespace pxt.blocks {
         images: 45,
         variables: 330,
         text: 160,
-        lists: 260
+        lists: 260,
+	math: 230,
```

Comment:
Oh, sorry. I will add another commit to change tabs to spaces. I am using vim on Linux.